calico/security/encrypt-cluster-pod-traffic.md

```diff
@@ -22,17 +28,21 @@ This how-to guide uses the following {{site.prodname}} features:
 
 ### Before you begin...
 
-The following platforms:
+**Unsupported**
+
+- GKE
+- Using your own custom keys to encrypt traffic
+
+**Limitations**
 
-- Kubernetes, on-premises
-- EKS using Calico CNI
-- EKS using AWS CNI
-- AKS using Azure CNI
+- IPv4 only
```

Comment:
This should be removed as we now support IPv6

Reply:
I thought IPv6/ipv4 support was only for host-to-host traffic. And that pod-to-pod was only IPv4. Check with Mike Stephens to verify. If true, then suggest this to be more clear:

**Limitations**

- ipv4 for pod-to-pod traffic
- ipv4/ipv6 for host-to-host traffic
- For EKS, TBD...if Sujeet says we support both, then delete this entry
- For AKS, only Azure CNI

Reply:
Sujeet says EKS with Calico CNI is not supported. Again, until we revisit "support" for OSS versus CE with PM, I suggest we stay aligned with CE.

Reply:
I would go with this for now and get this merged for the migration. I will ask Sujeet to confirm. (Sujeet has already confirmed we do not support EKS with Calico CNI in 3.15). 

**Limitations**

- Encryption for pod-to-pod traffic, ipV4 only
- Encryption for direct node-to-node communication (ipv4/ipv6) is supported only on EKS and AKS-managed clusters:
   - EKS, AWS CNI only
   - AKS, Azure CNI only